fragmentshandler/fragmentshandler/src/androidTest/java/com/microsoft/device/dualscreen/fragmentshandler/utils/SampleActivity.kt

```diff
@@ -65,30 +70,29 @@ class SampleActivity : FragmentActivity(), ScreenInfoListener {
         lastSavedInstanceState = savedInstanceState
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_sample)
+        registerWindowInfoFlow()
     }
 
-    override fun onConfigurationChanged(newConfig: Configuration) {
-        super.onConfigurationChanged(newConfig)
-        ScreenManagerProvider.getScreenManager().onConfigurationChanged()
-    }
-
-    override fun onResume() {
-        super.onResume()
-        ScreenManagerProvider.getScreenManager().addScreenInfoListener(this)
-    }
-
-    override fun onPause() {
-        super.onPause()
-        ScreenManagerProvider.getScreenManager().removeScreenInfoListener(this)
+    private fun registerWindowInfoFlow() {
+        val windowInfoRepository = windowInfoRepository()
```

Comment:
Love it! The usage is even simpler, and requires less setup than previous version!